src/ray/raylet/format/node_manager.fbs

```diff
@@ -37,7 +37,7 @@ enum MessageType:int {
   ExecuteTask,
   // Reconstruct or fetch possibly lost objects. This is sent from a worker to
   // a local scheduler.
-  ReconstructObjects,
+  NotifyBlocked,
   // For a worker that was blocked on some object(s), tell the local scheduler
   // that the worker is now unblocked. This is sent from a worker to a local
   // scheduler.
```

Comment:
Why does the worker need to notify the scheduler it's unblocked, can't the scheduler track if the worker is waiting on any objects?

Reply:
We had to do this in the legacy code because we didn't do very good dependency tracking in the backend. We technically don't have to anymore in the raylet codebase, but I think it's easier to leave it this way for now. I left this [TODO](https://github.com/ray-project/ray/blob/master/src/ray/raylet/node_manager.cc#L1173) but if you prefer that we do it in this PR, let me know.